video.h

```diff
@@ -7,7 +7,8 @@
 
 #include <stdbool.h>
 #include <stdint.h>
-#include <SDL2/SDL.h>
+#define SDL_MAIN_HANDLED
```

Comment:
I would put this define at the top of the `main.c` file (or wherever the `main` function is defined).
Otherwise reordering of includes can introduce a bug.

Reply:
Ooops. I meant to put it in `video.c`. I put it there instead of `main.c` because there's no reference to SDL in `main.c`. SDL initialization is made in `video.c`.

Reply:
Never mind, I was wrong. `main.c` includes `video.h` which includes `SDL.h`.